scripts/kubernetes/serversChart/templates/mpsDeployment.yaml

```diff
@@ -0,0 +1,158 @@
+#/*********************************************************************
+# Copyright (c) Intel Corporation 2020
+# SPDX-License-Identifier: Apache-2.0
+#**********************************************************************/
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: azure-mpsserver
+spec:
+  replicas: {{ .Values.mps.replicaCount }}
+  selector:
+    matchLabels:
+      app: azure-mpsserver
+  template:
+    metadata:
+      labels:
+        app: azure-mpsserver
+    spec:
+      imagePullSecrets:
+        - name: {{ .Values.images.pullSecret }}
+      containers:
+      - name: azure-mpsserver
+        image: {{ .Values.images.mps }}
+        env:
+          - name: "REACT_APP_MPS_SERVER"
```

Comment:
Are these still being used somewhere?

Reply:
I left them in to be backward compatible.. I can remove them.